src/Plugin.php

```diff
@@ -109,6 +107,11 @@ public static function init() {
     PlusProducts::init();
 
     if (is_admin()) {
+      if (function_exists('register_field_group')) {
+        // Adds optional customization of page title.
+        ProductAttributePageTitle::admin_init();
+      }
```

Comment:
Maybe we could move this to `Admin::init()`?

Reply:
Yes, good idea, it looks strange here.